pom.xml

```
@@ -6,14 +6,73 @@
     <groupId>fish.payara.advisor</groupId>
     <artifactId>advisor-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0</version>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Payara Advisor Tool</description>
+    <url>http://www.payara.fish</url>
+
+    <licenses>
+        <license>
+            <name>GPL Version 2 with Classpath Exception</name>
+            <url>https://github.com/payara/Payara/blob/master/LICENSE.txt</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git:github.com/payara/AdvisorTool.git</connection>
+        <developerConnection>git@github.com:payara/AdvisorTool.git</developerConnection>
+        <url>https://github.com/payara/AdvisorTool</url>
+        <tag>master</tag>
+    </scm>
+    
+    <developers>
+        <developer>
+            <id>breakponchito</id>
+            <name>Alfonso Valdez</name>
+            <organization>Payara Foundation</organization>
+            <organizationUrl>http://www.payara.org</organizationUrl>
+        </developer>
+    </developers>
 
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+        </snapshotRepository>
+    </distributionManagement>
+    
+    <profiles>
+        <profile>
+            <id>ossrh</id>
+            <properties>
+                <skipTests>true</skipTests>
+                <maven.test.skip>true</maven.test.skip>
```

Doesn't `maven.test.skip` imply `skipTests`?
```suggestion
                <maven.test.skip>true</maven.test.skip>
```

---

good catch